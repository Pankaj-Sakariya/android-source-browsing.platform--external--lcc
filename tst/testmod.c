extern void printf(const char* s,...);

void main(int argc, char** argv) {
    int i;
    for (i = 0; i < 12; i++) {
        printf("%d / %d = %d, %d %% %d = %d\n", i, 3, i / 3, i, 3, i % 3);
    }
}
