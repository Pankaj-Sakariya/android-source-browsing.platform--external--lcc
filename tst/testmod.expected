0 / 3 = 0, 0 % 3 = 0
1 / 3 = 0, 1 % 3 = 1
2 / 3 = 0, 2 % 3 = 2
3 / 3 = 1, 3 % 3 = 0
4 / 3 = 1, 4 % 3 = 1
5 / 3 = 1, 5 % 3 = 2
6 / 3 = 2, 6 % 3 = 0
7 / 3 = 2, 7 % 3 = 1
8 / 3 = 2, 8 % 3 = 2
9 / 3 = 3, 9 % 3 = 0
10 / 3 = 3, 10 % 3 = 1
11 / 3 = 3, 11 % 3 = 2
